README.md

```diff
@@ -1,5 +1,11 @@
 Installation
 ------------
 
-Just run `./install.sh` to copy the files into the right places. Feel free to
-submit a Pull Request featuring a cleaner way!
+For easy automatic installation you can install this plugin using the [Gedit Plugin Installer](https://github.com/lwindolf/gedit-plugininstaller/) or you can do it manually: 
```

Comment:
" or you can do it manually:" .. "For manual installation run" is a bit repetitive.

IMO just remove " or you can do it manually:"

Reply:
Also break the lines to be <= 80cpl.